sycl/plugins/cuda/pi_cuda.cpp

```diff
@@ -100,12 +100,13 @@ pi_result cuda_piEventRetain(pi_event event);
 } // extern "C"
 
 _pi_event::_pi_event(pi_command_type type, pi_context context, pi_queue queue)
-    : commandType_{type}, refCount_{1}, isCompleted_{false},
-      isRecorded_{false},
-      isStarted_{false}, event_{nullptr}, queue_{queue}, context_{context} {
+    : commandType_{type}, refCount_{1}, isCompleted_{false}, isRecorded_{false},
+      isStarted_{false}, evEnd_{nullptr}, evStart_{nullptr}, evQueued_{nullptr},
+      queue_{queue}, context_{context} {
 
   if (is_native_event()) {
-    PI_CHECK_ERROR(cuEventCreate(&event_, 0));
+    PI_CHECK_ERROR(cuEventCreate(&evQueued_, 0));
+    PI_CHECK_ERROR(cuEventCreate(&evEnd_, 0));
```

Comment:
Shouldn't all this stuff be done only if queue has "enable_profiling" property?

Reply:
Good point! However, we still need to create and record `evEnd_` as it is also used for synchronization.